### ISSUE TITLE
Autoreloader also fails if `six` is present, and `winreg` is not. On linux.`

### ISSUE THREAD

**Issue by bb-migration** [cherrypy/cherrypy#1341]
Originally reported by: **Fake Name (Bitbucket: [fake-name](http://bitbucket.org/fake-name), GitHub: [fake-name](http://github.com/fake-name))**

----------------------------------------

 It looks like something is wrong somewhere. The autoreloader is failing:  

```

[27/Sep/2014:22:59:15] ENGINE Bus STARTING
[27/Sep/2014:22:59:15] ENGINE Started monitor thread 'Autoreloader'.
[27/Sep/2014:22:59:15] ENGINE Started monitor thread '_TimeoutMonitor'.
[27/Sep/2014:22:59:15] ENGINE Serving on http://0.0.0.0:6543
[27/Sep/2014:22:59:15] ENGINE Bus STARTED
[27/Sep/2014:22:59:16] ENGINE Error in background task thread function <bound method Autoreloader.run of <cherrypy.process.plugins.Autoreloader object at 0x7f69e4de9b38>>.
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/cherrypy/process/plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.4/dist-packages/cherrypy/process/plugins.py", line 632, in run
    for filename in self.sysfiles() | self.files:
  File "/usr/local/lib/python3.4/dist-packages/cherrypy/process/plugins.py", line 616, in sysfiles
    hasattr(m, '__loader__') and
  File "/usr/lib/python3/dist-packages/six.py", line 116, in __getattr__
    _module = self._resolve()
  File "/usr/lib/python3/dist-packages/six.py", line 105, in _resolve
    return _import_module(self.mod)
  File "/usr/lib/python3/dist-packages/six.py", line 76, in _import_module
    __import__(name)
ImportError: No module named 'winreg'

Exception in thread Autoreloader:
Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.4/dist-packages/cherrypy/process/plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.4/dist-packages/cherrypy/process/plugins.py", line 632, in run
    for filename in self.sysfiles() | self.files:
  File "/usr/local/lib/python3.4/dist-packages/cherrypy/process/plugins.py", line 616, in sysfiles
    hasattr(m, '__loader__') and
  File "/usr/lib/python3/dist-packages/six.py", line 116, in __getattr__
    _module = self._resolve()
  File "/usr/lib/python3/dist-packages/six.py", line 105, in _resolve
    return _import_module(self.mod)
  File "/usr/lib/python3/dist-packages/six.py", line 76, in _import_module
    __import__(name)
ImportError: No module named 'winreg'

```

This is a Ubuntu 14.04 x64. `winreg` is a windows registry editing library.

From what I can tell, `six` is iterating over every value in `sys.modules`, and attempting to import each. See https://bitbucket.org/gutworth/six/issue/63/importing-six-breaks-pickling.

Basically, I think there needs to be a try/cach in `dist-packages/cherrypy/process/plugins.py`.

----------------------------------------
- Bitbucket: https://bitbucket.org/cherrypy/cherrypy/issue/1341

**Comment by webknjaz** [comment]
Closing as duplicate of #1319
